.github/workflows/test.yml

```diff
@@ -87,4 +52,22 @@ jobs:
           distribution: 'temurin'
           java-version: '17'
       - uses: gradle/actions/setup-gradle@v3
-      - run: .\gradlew.bat test
+      - name: Set gradle command
+        shell: bash
+        id: vars
+        run: |
+          if [[ "$RUNNER_OS" == "Windows" ]]; then
+              echo gradle_cmd=".\gradlew.bat" >> $GITHUB_OUTPUT
```

Comment:
I could be wrong, but I think you can simply set this at the top level, and `./gradlew` will work in all cases. This way you can simply change all of this to:

```
defaults:
  run:
    shell: bash

//...

  - name: Install JDK (Linux)
    if: runner.os == 'Linux'
    run: sudo apt-get install openjdk-17-jdk
  - name: Install JDK (macOS)
    if: runner.os == 'macOS'
    run: brew install microsoft-openjdk@17
  
  - name: ... tests
    run: ./gradlew test --tests "com.mlt.converter.encodings.*"

Reply:
Thanks! Stepping out for the day now, but will take a look at integrating this suggestion tomorrow.

Reply:
Excellent, it works. Thanks! https://github.com/maplibre/maplibre-tile-spec/pull/48/commits/92b16d781861c61abcc874bc109a773d75e2d6bc